my-sms/config/database.yml

```diff
@@ -0,0 +1,35 @@
+# MySQL.  Versions 4.1 and 5.0 are recommended.
+#
+# Install the MYSQL driver
+#   gem install mysql2
+#
+# Ensure the MySQL gem is defined in your Gemfile
+#   gem 'mysql2'
+#
+# And be sure to use new-style password hashing:
+#   http://dev.mysql.com/doc/refman/5.0/en/old-client.html
+
+common: &common
+  adapter: mysql2
+  encoding: utf8
+  reconnect: false
+  pool: 5
+  username: root
+  password: swordfish
```

Comment:
Environment variables are the most common approach when it comes to sharing (or not!) sensitive information such as passwords. Gradually being replaced by the use of Secrets in more modern versions of rails.